### DISCUSSION TITLE
Mayastor or Openebs

### DISCUSSION THREAD

**Discussion by uhlhosting** [openebs/openebs#3488] [Q&A] [answered]
Hi,

I am new to Mayastor, anyone can explain me what is better documentation to follow and or if there are both projects maintained, what is best to use mayastor or openebs?

**Comment by niladrih** [comment]
Hi @uhlhosting, you can find Mayastor documentation here: https://mayastor.gitbook.io/

**Reply by uhlhosting** [reply]
After 2 years, guess the reply isnt really helping guys, what you think? I waited for anyone to reply and not to realize what to do for 2 years?

**Reply by avishnu** [reply, marked as answer]
Hi @uhlhosting , OpenEBS is a portfolio of Kubernetes storage solutions - both replicated and localpv. Amongst them, Mayastor is the latest and most active solution for replicated storage. You should definitely try Mayastor, post your experience and questions on the openebs community slack - https://openebs.io/community.
